dhall/src/Dhall/Import.hs

```diff
@@ -451,11 +454,10 @@ localToPath prefix file_ = liftIO $ do
             return "/"
 
         Parent -> do
-            pwd <- Directory.getCurrentDirectory
-            return (FilePath.takeDirectory pwd)
+            return ".."
 
         Here -> do
-            Directory.getCurrentDirectory
+            return "."
 
```

Comment:
What was the rationale behind having `localToPath` return absolute paths in the first place?

Reply:
Never mind, I simply changed `exprFromUncachedImport` to call `makeAbsolute` on the resulting path, thus recovering the original behaviour.